src/app/content/components/AssignedTopBar.tsx

```diff
@@ -39,21 +42,60 @@ const StyledSectionTitle = styled.h2`
   `)}
 `;
 
+declare const window: any;
```

Comment:
I don't notice any custom properties so it seems like you wouldn't need it this.... maybe prerendering related?

Reply:
looks like an artifact of an earlier age